Audio-Transcription-Firefox/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+    "manifest_version": 2,
+    "name": "Audio Recorder",
```

Comment:
Do you think we should name this differently, since our focus is on live transcription?